.github/workflows/main.yml

```diff
@@ -541,24 +541,23 @@ jobs:
           test -f h2o2-test.ck
 
   windows-2022:
-    name: ${{ matrix.os }}, MSVC ${{ matrix.vs-toolset }}, Python ${{ matrix.python-version }}, fmt ${{ matrix.fmt-ver }}
-    runs-on: ${{ matrix.os }}
+    name: Windows 2022, Python ${{ matrix.python-version }}, fmt ${{ matrix.fmt-ver }}
```

Comment:
I think I had that matrix run set up to be flexible for various tests ... it's ok to simplify though.

Reply:
The comments on https://github.com/actions/runner-images/issues/9701 suggest that flexibility on VS version is gone for the foreseeable future.

The Windows version switch would be pretty easy to reintroduce once there's a `windows-2025` image, assuming we want to leave the rest of the workflow the same. Or perhaps we'd swap this with the set of jobs that use `windows-2019`, since GitHub will probably deprecate that image around that time.